torchgeo/datasets/landcoverai.py

```diff
@@ -2,31 +2,33 @@
 # Licensed under the MIT License.
 
 """LandCover.ai dataset."""
-
+import abc
 import glob
 import hashlib
 import os
 from functools import lru_cache
-from typing import Callable, Dict, Optional
+from typing import Any, Callable, Dict, List, Optional, cast
 
 import matplotlib.pyplot as plt
 import numpy as np
 import torch
 from matplotlib.colors import ListedColormap
 from PIL import Image
+from rasterio.crs import CRS
 from torch import Tensor
+from torch.utils.data import Dataset
 
-from .geo import NonGeoDataset
-from .utils import download_url, extract_archive, working_dir
+from .geo import NonGeoDataset, RasterDataset
+from .utils import BoundingBox, download_url, extract_archive, working_dir
 
 
-class LandCoverAI(NonGeoDataset):
-    r"""LandCover.ai dataset.
+class LandCoverAIBase(Dataset[Dict[str, Any]], abc.ABC):
+    r"""Abstract base class for LandCover.ai Geo and NonGeo datasets.
 
     The `LandCover.ai <https://landcover.ai/>`__ (Land Cover from Aerial Imagery)
```

Comment:
Can you update this URL to the new URL? I did this in https://github.com/microsoft/torchgeo/pull/1125/files#diff-cfa6063281a8e2b83b8fa6c9304b312bd95a6e2d29d206b4c4b062995cc6c8cf but it's probably easier to do it here.